lib/http.js

```diff
@@ -159,6 +151,13 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
   app.get(
     routes.meterUsage,
     cors(),
+    asyncHandler(async function(req, res, next) {
+      // get meter so its data can be used in the authz check
+      const {meterId: id} = req.params;
+      const record = await meters.get({id});
+      req.meterRecord = record;
+      process.nextTick(next);
```

Comment:
If we need to execute this callback on the next turn of the event loop can we have a comment explaining the reasoning?

Reply:
Added.